### PR TITLE
Refactor editMessage to check if the current user is the same user that created the message

### LINKED ISSUE #81: Edit someone else's message
**Describe the bug**
You can edit someone else's message under their name

**To Reproduce**
1. Start application
2. Go to '/web/myprofile/editmessages?id=1
    This will lead you to edit 'Jame SBonds 070' message. 
    If you change the message and submit, it will say that it was 'Jame SBond 070' who edited it.

**Screenshot**
image
image

### PR DESCRIPTION
**Pull request by altmannk**
### Related Issue(s)
Closes #81 

### Proposed Changes
Checks if the current user is the same user that created the message. 
Meaning that it’s no longer possible to go to ‘/web/myprofile/editmessage?id=1’ to edit ‘Jame SBond 070’ message, it will redirect back to ‘/web/myprofile’ if trying to enter edit mode for that message.

### CODE REVIEW

**Review by Frlundsten — APPROVED** [review]
Critical solution, nice!

**Review by Win-ther — APPROVED** [review]
image

File: src/main/java/se/iths/springbootgroupproject/controllers/WebController.java [hunk 1]
```diff
@@ -113,8 +113,14 @@ public String editUserProfile(@Valid @ModelAttribute("formData") EditUserFormDat
 
 
     @GetMapping("/myprofile/editmessage")
-    public String editMessage(Model model, @RequestParam("id") Long id) {
+    public String editMessage(Model model, @RequestParam("id") Long id, @AuthenticationPrincipal OAuth2User principal) {
         Message message = messageService.findById(id);
+        User currectUser = userService.findByGitHubId(principal.getAttribute("id"));
```
Comment (marcusju23) [inline src/main/java/se/iths/springbootgroupproject/controllers/WebController.java:118]: Minor typo? Can be corrected in another PR 👍

**Review by marcusju23 — APPROVED** [review]
Minor typo, approving because of critical bug 😄
